CONTRIBUTING.md

```diff
@@ -36,110 +39,144 @@ Signed-off-by: Jane Smith <jane.smith@example.com>
 1. In the local copy of your fork, navigate to the docs folder.
-`cd docs`
+   `cd docs`
 1. Create and checkout a new branch to make changes within
-`git checkout -b <my-changes>`
+   `git checkout -b <my-changes>`
 1. Edit/add documentation.
-`vi <specific page>.md`
+   `vi <specific page>.md`
 1. Run site locally to preview changes.
-`make site`
-* **Note:** *From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artefacts for production.*
+   `make site`
+
+- **Note:** _From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artefacts for production._
+
 1. Commit, [sign-off](#commit-signing), and push changes to your remote branch.
-`git push origin <my-changes>`
+   `git push origin <my-changes>`
 1. Open a pull request (in your web browser) against our main repo: https://github.com/layer5io/meshery.
 
-
 ## <a name="contributing-meshery">Meshery Contribution Flow</a>
+
 Meshery is written in `Go` (Golang) and leverages Go Modules. UI is built on React and Next.js. To make building and packaging easier a `Makefile` is included in the main repository folder.
 
-Relevant coding style guidelines are the [Go Code Review Comments](https://code.google.com/p/go-wiki/wiki/CodeReviewComments) and the _Formatting and style_ section of Peter Bourgon's [Go: Best 
+Relevant coding style guidelines are the [Go Code Review Comments](https://code.google.com/p/go-wiki/wiki/CodeReviewComments) and the _Formatting and style_ section of Peter Bourgon's [Go: Best
 Practices for Production Environments](https://peter.bourgon.org/go-in-production/#formatting-and-style).
 
-__Please note__: All `make` commands should be run in a terminal from within the Meshery's main folder.
+**Please note**: All `make` commands should be run in a terminal from within the Meshery's main folder.
 
 ### Prerequisites for building Meshery in your development environment:
+
 1. `Go` version 1.11+ installed if you want to build and/or make changes to the existing code.
 1. `GOPATH` environment variable should be configured appropriately
 1. `npm` and `node` should be installed on your machine, preferably the latest versions.
 1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have a version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
 
 #### Build and run Meshery server
+
 To build & run the Meshery server code, run the following command:
```

Comment:
The information from line: 106 can be moved here in a way like:
```Before building & runnig the Meshery server code, you need to setup the Meshery UI:
  <deps, export cmnd>, then the 
 Command to build & run Meshery server is,
 ```sh
 make run-local
```
```

Reply:
I've done suggested changes in a way it felt appropriate to me. If you want me to make further changes, please let me know, and I'll do so.